examples/aws/REAMDE.md

```diff
@@ -0,0 +1,36 @@
+# AWS Kubeflow Manifests
+
+## Overview
+
+In these manifest directories you can find example Kustomize manifests as well as instructions for installing Kubeflow components configured with supported AWS services.
+
+## Provisioning AWS Resources 
```

Comment:
change to pre-requisites?

And add EKS cluster, eksctl, kubectl, awscli, yq, kustomize

in EKS cluster point, we can have the instructions to create if they don't have one already

yq is needed in cognito